### ISSUE TITLE
Security/Interception

### ISSUE THREAD

**Issue by AlistairXX** [MickMake/Project-LoRaSMS#2]
These messages are traveling on a particular frequency if I'm aware of the frequency...then I can capture all of the packets of that frequency and read those messages. It's not like it has any kind of encryption like we have on mobile towers. 

The only possible way might be to write a code for Arduino to convert a message into a hash which can be deciphered by the receiving arduino-lora system. So even in case of Interception,the message might be safe. But power of Arduino is very less,so in case of encryption we have to set a same decipher for every message.What do you think?

**Comment by MickMake** [comment]
Absolutely! This was only just a PoC and there's a lot of work yet to be done before it can be used in trusted situations.

I would move to a BLE based MCU as these have onboard hardware encryption. Something like an nRF52. The battery life would also be a lot longer!

**Comment by WereCatf** [comment]
If you don't care about someone seeing the contents of the messages, only that no outsider can send unauthorized messages to your device (like e.g. "Turn the relay off"), the simplest and most efficient way would simply be using one-time pads. Not very suitable for this specific project here, but it's a good approach to keep in mind for the aforementioned stuff of controlling relays and such.
